larpandoracontent/LArCheating/CheatingRemovingCosmicRays.cc

```diff
@@ -0,0 +1,72 @@
+/**
+ *  @file   larpandoracontent/LArCheating/CheatingRemovingCosmicRays.cc
+ * 
+ *  @brief  Implementation of the cheating removing cosmic rays algorithm class.
+ * 
+ *  $Log: $
+ */
+
+#include "Pandora/AlgorithmHeaders.h"
+#include "Helpers/MCParticleHelper.h"
+
+#include "larpandoracontent/LArHelpers/LArMCParticleHelper.h"
+
+#include "larpandoracontent/LArCheating/CheatingRemovingCosmicRays.h"
+
+using namespace pandora;
+
+namespace lar_content
+{
+
+CheatingRemovingCosmicRays::CheatingRemovingCosmicRays() :
```

Comment:
Suggest removing default constructor entirely. (Or use = default in its prototype.) (Or add initialisation of additional third member variable.)